src/get-started/install/windows_quick.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Windows install
+description: How to install on Windows.
+short-title: Windows
+next:
+  title: Set up an editor
+  path: /get-started/editor
```

Comment:
Will this conflict with the existing [get-started/editor](https://docs.flutter.dev/get-started/editor) page?

Reply:
Good question. This PR needs to be staged to be sure.

Reply:
I misread this line, this is the `next` URL in the getting started flow.